lib/index.js

```diff
@@ -118,6 +117,15 @@ Builder.prototype.cacheTheme = function(result) {
 	});
 };
 
+/**
+ * Creates a themebuild
+ * @param {object} options
+ * @param {object} options.compiler compiler object as passed to less
+ * @param {boolean} options.compiler.cssVars whether or not to enable css variables output
```

Comment:
`options.cssVars` (see comment in readme)

Reply:
Ok